src/browser/NativeMessagingHost.cpp

```diff
@@ -142,6 +142,14 @@ void NativeMessagingHost::newLocalConnection()
 void NativeMessagingHost::newLocalMessage()
 {
     QLocalSocket* socket = qobject_cast<QLocalSocket*>(QObject::sender());
+    if (socket) {
```

Comment:
Just put this code after the if (!socket....) check on line 154, no need for additional checks